.gitignore

```diff
@@ -34,3 +34,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+/.idea
```

Comment:
@StefanGajic well this will ignore future changes to that particular directory it won't remove the directory. You have to do `git rm` to actually remove the directory from the repo.